berg.yaml

```diff
@@ -41,7 +41,29 @@ update:
 
 test:
   pipeline:
-    # AUTOGENERATED
-    - runs: |
+    - name: Version and help check
+      runs: |
         berg --version
         berg --help
+    - name: Test command parsing
+      runs: |
+        # Test invalid command returns proper error code
+        if berg nonsense-command 2>/dev/null; then
+          echo "Invalid command should have failed"
+          exit 1
+        fi
+
+        # Verify help command lists available commands
+        berg help | grep -q "Commands:" || exit 1
+    - name: Test version output format
+      runs: |
+        # Check version output format and value
+        berg --version | grep -q "codeberg-cli ${{package.version}}" || exit 1
```

Comment:
These are very, very brittle, and ill-advised.  We've long had the ability to test version strings in this way (including in the original non-AI generate tests), but we've found that these break for simple, dumb, and benign reasons, and create toil on the sustaining team....